File: Python/Program/Periferal.py
from Komponen import Komponen

class Periferal(Komponen):
    def __init__(self, merk, nama, koneksi):
        super().__init__(merk, nama)
        self.koneksi = koneksi
        
    def set_koneksi(self, koneksi):
        self._koneksi = koneksi
        
    def get_koneksi(self):
        return self._koneksi

File: Java/Program/Periferal.java
public class Periferal extends Komponen {
    private String koneksi;

    public Periferal(String merk, String nama, String koneksi) {
        super(merk, nama);
        this.koneksi = koneksi;
    }

    public void setKoneksi(String koneksi) {
        this.koneksi = koneksi;
    }

    public String getKoneksi() {
        return koneksi;
    }
}